app/models/user.rb

```diff
@@ -33,11 +33,79 @@ class User < ApplicationRecord
 
   scope :remote_lawyers, -> { where(remote_clinic_lawyer: true) }
 
+  filterrific(
+    default_filter_params: { sorted_by: 'created_at_desc' },
+    available_filters: %i[
+      filter_first_name
+      filter_last_name
+      filter_email
+      with_volunteer_type
+      sorted_by
```

Comment:
I'm wondering if it would be more consistant to prepend all the filterrific scopes with `filter_`? Thoughts?

Reply:
Yea! That's an easy one. Will do.